src/lib/types/group.ts

```diff
@@ -33,6 +33,11 @@ export interface IGroupModel extends IGroup {
     projects?: string[];
 }
 
+export interface ICreateGroupModel extends IGroup {
```

Comment:
new TS model reflecting openapi schema change. Users are optional and each user is represented by id only